wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-container fluid fill-height justify-center>
+    <v-card :loading="getData" width="80%">
+      <v-card-title>
+        <h1 class="display-1">Game Words</h1>
+      </v-card-title>
+      <v-card-text>
+        <v-row>
+          <v-col>
+            <v-text-field
+          v-model="wordFilter"
+          append-icon="mdi-magnify"
+          label="Search"
+          single-line
+          hide-details
+        ></v-text-field>
+          </v-col>
+          <v-col>
+            <v-text-field label="Add a new word" v-model="newWord"></v-text-field>
+          </v-col>
+          <v-col>
+            <v-btn @click="addWord()">Add Word</v-btn>
+          </v-col>
+        </v-row>
+          
+        <v-simple-table loading>
+          <thead>
+            <tr>
+              <th>Word</th>
+              <th>Playable</th>
+              <th>Delete</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="(stat, index) in stats" :key="index">
+              <td>{{ stat.value }}</td>
+              <td>
+                <v-checkbox
+                v-model="stat.common"
+                color="red"
+                @click="changeCommon(stat.value, stat.common)"
+              ></v-checkbox></td>
+              <td> 
+                <v-btn icon> 
+                    <v-icon small> mdi-delete </v-icon>
+                </v-btn>
+              </td>
+            </tr>
+          </tbody>
+        </v-simple-table>
+      </v-card-text>
+      <v-card-actions>
+          <v-pagination
+        v-model="currentPage"
+        :length="totalPages"
+        @input="accessPage"
+      ></v-pagination>
+      </v-card-actions>
+    </v-card>    
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator'
+@Component({})
+export default class WordEditor extends Vue {
+  
+  currentPage: number= 1
+
+  stats: any = []
+  getData: boolean = true
+  pageNum: number = 1
+  wordsPerPage: number = 20
+  totalPages: number = 10
+  wordFilter: string = "a"
+  newWord = ""
+
+  mounted() {
+    this.getWords();
+  }
+
+  @Watch("wordFilter")
+    getWords(){
+         var wordsPerPage = this.wordsPerPage
+    var pageNum = this.pageNum
+    var wordFilter = this.wordFilter
+    this.getData = true;
+    this.$axios.get('/api/Word/GetWordsPerPage', { params: { wordsPerPage : wordsPerPage, pageNum : pageNum, wordFilter : wordFilter } }).then((response) => {
```

Comment:
one thing that we did was had a check on the word filter, and if it was empty, it used a different api call that did not require a search parameter. That way if the user entered a word in search and then emptied it, there were not weird issues.